hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ServiceInternalProcessingTimeEnricher.java

```diff
@@ -75,6 +76,16 @@ public void enrichTrace(StructuredTrace trace) {
                             entryApiBoundaryEventDuration
                                 - totalEdgeDurations
                                 - httpExitCallsSum)));
+            // also put into metric map
+            entryApiBoundaryEvent
+                .getMetrics()
+                .getMetricMap()
+                .put(
+                    EnrichedSpanConstants.INTERNAL_SVC_LATENCY,
+                    MetricValue.newBuilder()
```

Comment:
Please use the `fastNewBuilder` method of the `AvroBuilderCache` class instead of newBuilder().